### PR TITLE
Fix use of uploads directory on local server

### PR DESCRIPTION
**Pull request by roborourke**
After the change to the cloud module to ensure media paths matched the site's host name it broke local server's assumption that the uploads path was at `s3-<project>.altis.dev`.

This adds a 301 redirect on the nginx server to map to the correct URL on the S3 container.

### CODE REVIEW

File: docker/docker-compose.yml [hunk 1]
```diff
@@ -114,6 +114,9 @@ services:
       - "traefik.port=8080"
       - "traefik.protocol=https"
       - "traefik.docker.network=proxy"
+      - "traefik.uploads.frontend.redirect.regex=^https?://(.*?).altis.dev/uploads/(.*)"
+      - "traefik.uploads.frontend.redirect.replacement=https://s3-${COMPOSE_PROJECT_NAME:-default}.altis.dev/uploads/$$2"
```
Comment (rmccue) [inline docker/docker-compose.yml:118]: Should this use $$1 instead?

Comment (roborourke) [reply docker/docker-compose.yml:118]: no, I made that error first time round - there are 2 capturing brackets

Comment (rmccue) [reply docker/docker-compose.yml:118]: Sorry, I mean instead of the environment variable, not instead of the path replacement

Comment (roborourke) [reply docker/docker-compose.yml:118]: No, it's intentional. The S3 container URL will always be that

**Review by rmccue — APPROVED** [review]

File: docker/docker-compose.yml [hunk 2]
```diff
@@ -114,6 +114,10 @@ services:
       - "traefik.port=8080"
       - "traefik.protocol=https"
       - "traefik.docker.network=proxy"
+      # Redirect any requests to /uploads to the S3 container frontend.
+      - "traefik.uploads.frontend.redirect.regex=^https?://(?:.*?).altis.dev/uploads/(.*)"
```
Comment (rmccue) [inline docker/docker-compose.yml:118]: Just to check, this won't cause an infinite redirect right??

Comment (roborourke) [reply docker/docker-compose.yml:118]: I did test it 😛  - the traefik rules apply first before the request gets to the container so `s3-*.altis.dev` requests are never handled by nginx.

Comment (joehoyle) [reply docker/docker-compose.yml:118]: WHen this says "redirect" does it mean HTTP redirect? If so, is this just used for legacy URLs? Just want to make sure we are not always issuing 301 redirects!

**Pull request merged**